test/ui-test/testSuites/suite_status/tst_searchFlows/test.feature

```diff
@@ -0,0 +1,57 @@
+#******************************************************************************
+# Status.im
+#*****************************************************************************/
+#/**
+# * \file   test.feature
+# *
+# * \test   Status Desktop - Search flows
+# * \date   August 2022
+# **
+# *****************************************************************************/
+
+Feature: Search feature (ctrl+F)
+
+    As a user, I want to search the different chats and messages of the app
+
+    Covers the search flows
+
+    Background:
+        Given A first time user lands on the status desktop and generates new key
+        When user signs up with username tester123 and password TesTEr16843/!@00
+        Then the user lands on the signed in app
+        When user joins chat room search-automation-test-1
+        When user joins chat room search-automation-test-2
```

Comment:
Do you want to add these 2 whens?? If yes, maybe it is better to change the second one to and `and` to have a better readability